packages/components/Checkbox/src/Checkbox.settings.windows.ts

```diff
@@ -1 +1,95 @@
-export * from './Checkbox.settings.old';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import { checkboxName, ICheckboxType } from './Checkbox.types';
+
+export const checkboxSelectActionLabel = 'Toggle the Checkbox';
+
+export const settings: IComposeSettings<ICheckboxType> = [
```

Comment:
Maybe add a comment in the file that the settings differ because windows doesn't support SVG yet